WcaOnRails/app/views/registrations/edit_registrations.html.erb

```diff
@@ -53,45 +60,41 @@
               </td>
 
               <td class="wca-id">
-                <% if registration.wca_id %>
-                  <%= render "shared/wca_id", wca_id: registration.wca_id %>
-                <% end %>
+                <%= wca_id_link registration.wca_id %>
               </td>
 
               <td class="name"><%= registration.name %></td>
               <td class="countryId"><%= registration.country.name %></td>
               <td class="birthday"><%= registration.birthday %></td>
-              <% @competition.events.each do |event| %>
-                <td>
-                  <% if registration.events.include?(event) %>
-                    <span title="<%= event.name %>"
-                          class="cubing-icon event-<%= event.id %>"
-                          data-toggle="tooltip"
-                          data-placement="bottom"
-                          data-container="body">
-                    </span>
-                  <% end %>
-                </td>
+              <% if @show_events %>
```

Comment:
Ah yes! The 3rd one is in `WcaOnRails/app/views/registrations/_edit_registrations_table_footer.html.erb`.